check_pgactivity

```diff
@@ -5551,6 +5558,18 @@ sub check_streaming_delta {
 
     @rs = @{ query_ver( $hosts[0], %queries ) };
 
+    my $index = 0;
```

Comment:
I thought the point of this patch was to avoid having false positive warning for outdated streaming connections due to pg_basebackup.

Having the "#of slaves" with the excluded connections allows to monitor if you exhaust the total number of possible connections for instance, and possibly know when/how long temporary use of streaming connections happened.